app/client/components/pages/singleBulkDisbursementPage.jsx

```diff
@@ -89,8 +93,19 @@ class SingleBulkDisbursementPage extends React.Component {
 
     let status = bulkItem && bulkItem.state;
     let transferType = bulkItem && bulkItem.transfer_type;
-    let createdBy = bulkItem && bulkItem.creator_email;
+    let creator_user = bulkItem && bulkItem.creator_user;
+    let approvers = (bulkItem && bulkItem.approvers) || [];
     let label = bulkItem && bulkItem.label;
+    const approversList = approvers.map(approver => {
+      return (
+        <a
+          style={{ cursor: "pointer" }}
+          onClick={() => this.navigateToUser(approver && approver.id)}
+        >
+          {approver && " " + approver.email + " "}
+        </a>
+      );
+    });
```

Comment:
Nit: I think this would be nice if there was a comma or separator between emails